frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/video/upload_video_menu.dart

```diff
@@ -86,6 +94,10 @@ class _EmbedUrlState extends State<_EmbedUrl> {
       return widget.onSubmit(inputText);
     }
 
+    if (inputText.startsWith('https://youtube')) {
```

Comment:
`^(https?:\/\/)?(www\.)?(youtube\.com|youtu\.be)\/`

use regex would more accurately. for example, here're some edge case
1. youtube
2. www.youtube
3. youtu.be